snuba/datasets/storages/errors.py

```diff
@@ -69,9 +64,7 @@
         ("retention_days", UInt(16)),
         ("deleted", UInt(8)),
```

Comment:
We are likely going to need to turn this into a real version column. 
I think we should rename it to row_version (version already exists), or soemething else while we are rebuilding the table.

Reply:
I'll go ahead and rename it - two questions
1 - Is UInt8 sufficient or do you want to increase the size?
2 - What value do you want the replacer to write for a deletion?

Reply:
@fpacifici I went ahead with writing 255 instead of 1 for deletions - does that work for you?